ui/app/AppLayouts/Onboarding2/pages/BackupSeedphraseKeepOrDelete.qml

```diff
@@ -0,0 +1,48 @@
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Layouts
+
+import StatusQ.Core
+import StatusQ.Components
+import StatusQ.Controls
+import StatusQ.Core.Theme
+
+import AppLayouts.Onboarding2.components
+
+OnboardingPage {
+    id: root
+
+    readonly property bool removeSeedphrase: cbAck.checked
+
+    title: qsTr("Choose keep or delete recovery phrase")
```

Comment:
I see that this comes from the design, but this sounds weird, since it's not valid English.

Maybe we could just call it `Keep or delete your recovery phrase`?  cc @xAlisher